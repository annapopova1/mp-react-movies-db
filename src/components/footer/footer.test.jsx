import React from 'react';
import Footer from './footer';

describe('<Footer/>', () => {
  test('should render Footer component', () => {
    const footer = shallow(<Footer />);
    expect(footer).toMatchSnapshot();
  });
});
