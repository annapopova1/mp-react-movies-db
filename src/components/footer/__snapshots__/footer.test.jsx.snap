// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> should render Footer component 1`] = `
<footer
  className="container row navbar navbar-dark bg-dark py-1 footer"
>
  <span
    className="navbar-brand"
  >
    Movies DB, 2018
  </span>
</footer>
`;
